lib/buffer.js

```diff
@@ -65,6 +71,9 @@ function alignPool() {
  * would ever actually be removed.
  **/
 function Buffer(arg, encodingOrOffset, length) {
+  if (!(this instanceof Buffer)) {
+    bufferDeprecationWarning();
+  }
```

Comment:
Could you clarify the purpose of such change? It would introduce needless boilerplate and make it inconsistent with the rest of node.